src/install.js

```diff
@@ -56,11 +57,19 @@ export const DEPENDENCY_FIELDS = [
  * @param  {String} name - name of the dependency.
  * @return {Observable} - observable sequence of `package.json` objects.
  */
-export function resolveLocal (nodeModules, parentTarget, name) {
+export function resolveLocal (nodeModules, parentTarget, name, version) {
 	const linkname = path.join(nodeModules, parentTarget, 'node_modules', name)
 	const fetch = () => EmptyObservable.create()
 	log(`resolving ${linkname} from node_modules`)
 
+	// support `file:` with symlinks
+	if (version.substr(0, 5) === 'file:') {
```

Comment:
I've used it for the remote resolving but I guess having this simple test here is good enough since adding `npa` here would negatively impact performance.